test-case-reporting/test/test_result_record.test.ts

```diff
@@ -316,4 +316,12 @@ describe('TestResultRecord', () => {
       expect(testRuns).toMatchObject(sampleTestRuns);
     });
   });
+
+  describe('getTestRunsOfBuild', () => {
+    it.todo('gets only the test runs of the build');
```

Comment:
What does it return for a build that isn't in the DB? Same for below

Reply:
It'll return an empty list -- I don't think the view needs to distinguish between [builds that aren't in the database] and [builds that are in the database without test runs].

Done adding test todos for those cases.